doc/stdlib/math.rst

```diff
@@ -666,6 +666,18 @@ Mathematical Functions
 
    ``round(x, digits)`` rounds to the specified number of digits after the decimal place, or before if negative, e.g., ``round(pi,2)`` is ``3.14``. ``round(x, digits, base)`` rounds using a different base, defaulting to 10, e.g., ``round(pi, 1, 8)`` is ``3.125``.
 
+.. function:: round(x, RoundingMode)
+
+   Returns the nearest integral value of the same type as ``x`` to ``x``,
+   breaking ties using the specified :obj:`RoundingMode`.
+
+.. function:: round(z, RoundingMode1, RoundingMode2)
```

Comment:
I think the signature should rather be 

`round(z, RoundingModeReal, RoundingModeImaginary)`